### DISCUSSION TITLE
Unknown purl type: openvsx

### DISCUSSION THREAD

**Discussion by markodraisma** [nvim-java/nvim-java#107] [Q&A]
After installing the nvim-java plugin with Lazy for neovim, mason.nvim gives me two errors. Both for java-debug-adapter and java-test it says "Unknown purl type: openvsx". How can I solve this?

Regards, Marko

**Comment by s1n7ax** [comment]
https://github.com/nvim-java/nvim-java/wiki/Q-&-A#no_entry-unknown-purl-type-openvsx
